wherehows-web/app/routes/application.js

```diff
@@ -6,7 +6,8 @@ const {
   Route,
   run,
   get,
-  inject: { service }
+  inject: { service },
+  testing,
```

Comment:
I'm keeping the functionality the same as the ESA code: https://github.com/simplabs/ember-simple-auth/blob/master/addon/mixins/application-route-mixin.js#L135

I don't think that would make a difference in this case? @justinpark